db/migrate/20230501161349_create_income_limits_tables.rb

```diff
@@ -0,0 +1,90 @@
+class CreateIncomeLimitsTables < ActiveRecord::Migration[6.1]
+  def change
+    create_table :std_counties do |t|
+      t.string :name, null: false
+      t.integer :countynumber, null: false
```

Comment:
**Issue (blocking):** This is an example of one of the attributes that should be written out in snake case as `:county_number`.

Reply:
We pulled the table names from the CSV unaltered thinking that would make porting the data in a bit easier, but now that I'm looking at it again I agree. Will modify and clean up